scss/boost_union/post.scss

```diff
@@ -917,3 +914,994 @@ body.pagelayout-login:not(.loginbackgroundimage) #footnote {
+                    p {
+                        margin-bottom: 0;
+                    }
+                }
+                /* Padding and Position added for the "help icon" in menu */
+                .menu-helpicon {
+                    text-align: center;
+                    padding-right: 10px;
+                    margin-bottom: 10px;
+                    /* Margin removed in the menu description help icon in the menu */
+                    i {
+                        margin: 0;
+                    }
+                }
+                /* The font size and opacity reduced when the Menu link is selected as "Heading" option in the settings */
+                .dropdown-item {
+                    /* Dropdown menu link alignment at top */
+                    align-items: baseline;
+                    /* The "Tick" icon removed when the menu link is clicked */
+                    &:before {
+                        display: none;
+                    }
+                    &.menu-item-heading {
+                        font-size: 14px;
+                        opacity: .6;
+                        /* Menu link icon font size when the menu link as "Heading" and "icon" option selected in the settings */
+                        i.icon {
+                            font-size: 14px;
+                        }
+                    }
+                }
+                /* The below styles are header dropdown submenu  */
+                .dropdown-submenu {
+                    /* Position relative submenu link to control the arrow icon when it has submenus */
+                    > a.dropdown-item {
+                        position: relative;
+                        /* Header dropdown submenu */
+                        &:before {
+                            content: '';
+                        }
+                        /* The arrow icon and its postion at the right side is added when the header menu which has dropdown */
+                        &:after {
+                            content: '\f105';
+                            font-family: fontawesome;
+                            line-height: 10px;
+                            position: absolute;
+                            top: 40%;
+                            right: 10px;
+                        }
+                    }
+                    /* The submenu dropdown is hidden before the menu is clicked */
+                    ul {
+                        display: none;
+                    }
+                    /* The position relative is added for the dropdown menu */
+                    &.show {
+                        position: relative;
+                        /* The dropdown submenu style is added for the header menu */
+                        /* Height added to control the menu with scrollbard, border, border radius, background color, padding and its position styles are
```

Comment:
Do break the row earlier not to exceed 132 characters in length.

Reply:
Thanks, Luca.
I fixed this in a review commit now.